scripts/run-benchs.mjs

```diff
@@ -0,0 +1,38 @@
+/**
+ * @fileoverview This script is used to run all the benchmarks in the project.
+ * @link https://github.com/RafaelGSS/bench-node
+ */
+import { spawnSync } from 'node:child_process';
+import { styleText } from 'node:util';
+import { globSync } from 'node:fs';
+
+// hacky way to use work around npm workspaces
+const args = process.argv.slice(2);
+const w =
+	args
+		.find((arg) => arg.startsWith('--w='))
+		?.replace('--w=', '')
+		.trim()
+		.replace('package/', '') ?? '';
+
+if (!w.endsWith('/') && w.length > 0) w.concat('/');
+
+const files = globSync(`packages/${w}**/**.bench.{js,mjs}`);
+
+if (files.length === 0) {
+	console.log(`${styleText(['red'], '✕')} No benchmarks found`);
+} else {
```

Comment:
I think this should error

```suggestion
if (files.length === 0) {
	new Error(`${styleText(['red'], '✕')} No benchmarks found`);
}
```

And then there is no need to wrap the rest in `else`